chat-components/src/main/java/com/pubnub/components/chat/viewmodel/message/MessageViewModel.kt

```diff
@@ -123,14 +124,18 @@ class MessageViewModel constructor(
      * @return Flow of Message UI Paging Data
      */
     fun getAll(
+        filter: Query? = null,
```

Comment:
@kazydek filter and sort method added, to allow customer use it on database level